src/data/real/basic.lean

```diff
@@ -286,7 +286,9 @@ noncomputable instance decidable_lt (a b : ℝ) : decidable (a < b) := by apply_
 noncomputable instance decidable_le (a b : ℝ) : decidable (a ≤ b) := by apply_instance
 noncomputable instance decidable_eq (a b : ℝ) : decidable (a = b) := by apply_instance
 
-open rat
+/-- Show the underlying cauchy series for real numbers. -/
```

Comment:
I think we should be more careful to emphasise that the choice of representation is quite arbitrary.
```suggestion
/-- Show an underlying cauchy sequence for real numbers.

The representative chosen is the one passed in the VM to `quot.mk`, so two cauchy sequences
converging to the same number may be printed differently.
-/
```